packages/react/src/components/Pagination/next/Pagination.js

```diff
@@ -0,0 +1,421 @@
+/**
```

Comment:
I believe this file was removed in https://github.com/carbon-design-system/carbon/pull/12115